jekyll/_ccie/Artifactory.md

```diff
@@ -0,0 +1,61 @@
+# Artifactory 
+
+## Deploy
+
+Artifactory has great documentation explaining how to leverage their [REST API](https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API).
+
+We figure we use this space to highlight some sample projects that will show you how to best leverage using CircleCI and Artifactory together. 
+
+It is super important for you to have created your Repository before starting this example, otherwise CircleCI will have nowhere to store your dependencies.
+
+## JFrog CLI
+If you want to use the JFrog CLI you can install it by adding the following to your circle.yml.
```

Comment:
"If you want to use the JFrog CLI, you can install it by adding the following to your `circle.yml`:"